Documentation/ceph-csi-drivers.md

```diff
@@ -117,49 +151,24 @@ kubectl exec -ti -n rook-ceph rook-ceph-mon-a-6c4f9f6b6-rzp6r -- bash -c "ceph -
 
 Then create a Secret using admin and `kubernetes` keyrings:
 
-## rbd-secret.yaml
+In [secret](../cluster/examples/kubernetes/ceph/csi/example/rbd/secret.yaml), you
+need your Ceph admin/user password encoded in base64. Run `ceph auth ls` in one
+of your Ceph pods, encode the key of your admin/user and replace
```

Comment:
Perhaps we can be more precise, e.g:

"to encode your key, run `echo KEY | base64`."